src/Tribe/Aggregator/Cron.php

```diff
@@ -19,7 +19,7 @@ class Tribe__Events__Aggregator__Cron {
 	 * Limit of Requests to our servers
 	 * @var int
 	 */
-	private $limit = 5;
+	private $limit = 25;
```

Comment:
This could be problematic, depending on the server. I wonder why specifically we are changing this.

Reply:
With batch pushing we are increasing the number of requests back to the server with a regular set of imports we only have a single call and the delivery is processed on the client, in this case if a user has several schedule imports with batch pushing the client will increase the number of calls back to EA server if we set this as it is is likely that those imports would fail with an error due the number of requests, right now we remove this limit on loxi due this scenario with batch pushing that was creating a problem when pulling data from EA server (batch pushing)

Reply:
That's correct. 

However this can be filterable otherwise users can change it via the same filter, and batch imports is not specif to an option of imports as it can be available only for a set of imports, we can enable it dynamically before executing a batch import or processing the batch import, however I think the type of logic might be a bit overkill if eventually all imports are going to rely on batch pushing imports.

Reply:
Makes sense, thanksn.